### ISSUE TITLE
Read only mode

### ISSUE THREAD

**Issue by samuelbenhamou** [kairosdb/kairosdb#352]
It would be great to have an option to dedicate instances to read only from the Cassandra database
This would allow to have dedicated nodes for the data insertion while the graphs rendering is handled by different nodes.

**Comment by brianhks** [comment]
This has been implemented in 1.2.  There is a type designation that lets you specify if a server is ingest or a query node.

**Comment by hippoandy** [comment]
Dear Author,

I'm a user of KairosDB. I am wondering how to specify the KairosDB to read-only mode. Is the "Query" option only for "Select"?

I have a cluster with a Master and Slave.
On the Master, I already booted up the KairosDB, and it created the required keyspace for storing.

For the Slave, I tried to assign a Cassandra user with only select permission on the KairosDB default keyspace. But the KairosDB on the Slave node failed to start because it still requires "create" permission.

If I grant the select and create to that account, the KairosDB is able to boot up. However, in the log, the KairosDB will have an error saying that the assigned Cassandra account didn't have "modify" permission. This situation makes me feel confused.

Thanks in advance!
